packages/world/gas-report.json

```diff
@@ -129,13 +129,13 @@
     "file": "test/World.t.sol",
     "test": "testSetRecord",
     "name": "Write data to the table",
-    "gasUsed": 36896
+    "gasUsed": 39039
   },
```

Comment:
do we care that this test jumped ~2k gas due to changing this to use StoreSwitch's stored address?

If so, I could warm the store address or do two writes (one cold, one warm), but not sure if it's a thing we're concerned with benchmarking in this context